src/main/java/org/javarosa/xform/parse/XFormParser.java

```diff
@@ -1841,6 +1850,24 @@ private DataBinding processStandardBindAttributes(List<String> usedAtts, List<St
         "saveIncomplete"
     ));
 
+    private String toXMLTag(Element element) {
```

Comment:
So this method is not used anymore, right? You've replace it with org.javarosa.xform.util.XFormSerializer.elementToString, haven't you? If so, nuke it!

Reply:
Ouch! I thought I removed it.